strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -271,4 +273,16 @@ MavenArtifactRequestInterceptor mavenArtifactRequestInterceptor(RepositoryPathRe
     {
         return new MavenArtifactRequestInterceptor(repositoryPathResolver);
     }
+
+    @Override
+    public void addArgumentResolvers(List<HandlerMethodArgumentResolver> argumentResolvers)
+    {
+        argumentResolvers.add(repositoryInstanceMethodArgumentResolver());
+    }
+
+    @Bean
+    public RepositoryMethodArgumentResolver repositoryInstanceMethodArgumentResolver()
```

Comment:
`repositoryInstanceMethodArgumentResolver` it should be renamed to match `RepositoryMethodArgumentResolver `